dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapperTest.java

```diff
@@ -133,22 +133,26 @@ class EventOperationParamsMapperTest {
 
   @BeforeEach
   public void setUp() {
-    user = new User();
-    when(currentUserService.getCurrentUser()).thenReturn(user);
-
     orgUnit = createOrgUnit("orgUnit", orgUnitId);
     orgUnit.setChildren(
         Set.of(
             createOrgUnit("captureScopeChild", "captureScopeChildUid"),
             createOrgUnit("searchScopeChild", "searchScopeChildUid")));
+
+    user = new User();
+    user.setOrganisationUnits(Set.of(orgUnit));
+    when(currentUserService.getCurrentUser()).thenReturn(user);
   }
 
   @Test
   void shouldFailWithForbiddenExceptionWhenUserHasNoAccessToProgramStage() {
     programStage = new ProgramStage();
     programStage.setUid("PlZSBEN7iZd");
     EventOperationParams eventOperationParams =
-        EventOperationParams.builder().programStageUid(programStage.getUid()).build();
+        EventOperationParams.builder()
+            .programStageUid(programStage.getUid())
+            .orgUnitMode(ACCESSIBLE)
```

Comment:
Do we need to set this as it cannot be null and this test is testing the mapper. So the mapping in the web has not happened which would set the default?

Would it still make sense to add a default to the builder?

Reply:
Yes, that's the reason why I need to set the org unit mode in each particular test.